### PR TITLE
Document engine_getBlobsV1

### LINKED ISSUE #1700: Documentation of engine_getBlobsV1
Docs update required for [merged PR](https://github.com/hyperledger/besu/pull/7553)

### PR DESCRIPTION
**Pull request by alexandratran**
## Description

Document `engine_getBlobsV1` API method.

### Issue(s) fixed

Fixes #1700

### Preview

https://besu-docs-m0jg9ncly-hyperledger.vercel.app/public-networks/reference/engine-api#engine_getblobsv1

### CODE REVIEW

File: docs/public-networks/reference/engine-api/index.md [hunk 1]
```diff
@@ -224,6 +224,64 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
 
 </Tabs>
 
+### `engine_getBlobsV1`
+
+Returns the blobs corresponding to the specified blob versioned hashes.
+
+#### Parameters
+
+`blobHashes`: *array* of *strings* - Array of blob versioned hashes
```
Comment (pinges) [inline docs/public-networks/reference/engine-api/index.md:233]:
```suggestion
`versionedHashes`: *array* of *strings* - Array of blob versioned hashes
```

File: docs/public-networks/reference/engine-api/index.md [hunk 2]
```diff
@@ -224,6 +224,64 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
...
+
+#### Returns
+
+`engineGetBlobsResultV1`: *array* of *objects* - Array of blob and proof objects
```
Comment (pinges) [inline docs/public-networks/reference/engine-api/index.md:237]: Do we want to mention that for versioned hashes where the blob was not available besu responds with a null at that index of the response? In the spec they say this:
* result: `Array of BlobAndProofV1` - Array of [`BlobAndProofV1`](#BlobAndProofV1), items of which may be `null`.

File: docs/public-networks/reference/engine-api/index.md [hunk 3]
```diff
@@ -224,6 +224,64 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
...
+
+<Tabs>
+
+<TabItem value="curl HTTP" label="curl HTTP" default>
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"engine_getBlobsV1","params":[["0x000657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014"]],"id":1}' http://127.0.0.1:8550
```
Comment (pinges) [inline docs/public-networks/reference/engine-api/index.md:244]: Maybe add a second versioned hash, so it's obvious that this is a list?
Have you tested the curl command? Probably returns null, but should not complain.

File: docs/public-networks/reference/engine-api/index.md (hunk 2 above)
Comment (pinges) [reply docs/public-networks/reference/engine-api/index.md:237]: In our other API documentation do we use the 'official' names of the parameters and results? The result elements are called BlobAndProofV1.

Comment (alexandratran) [reply docs/public-networks/reference/engine-api/index.md]: For the type, using the generic "array of objects" is a little easier to understand. I've added a section detailing the blob and proof object for more information.

File: docs/public-networks/reference/engine-api/index.md [hunk 4]
```diff
@@ -224,6 +224,67 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
...
+
+`engineGetBlobsResultV1`: *array* of *objects* - Array of [blob and proof objects](objects.md#blob-and-proof-object),
+items of which can be `null`
+
+<Tabs>
+
+<TabItem value="curl HTTP" label="curl HTTP" default>
+
+```bash
+curl -X POST --data '{"jsonrpc":"2.0","method":"engine_getBlobsV1","params":[["0x000657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014","0x01f3a5c9d8e7b6a4c2d1e0f9b8a7c6d5e4f3a2b1c0d9e8f7b6a5c4d3e2f1a0b9"]],"id":1}' http://127.0.0.1:8550
```
Comment (pinges) [inline docs/public-networks/reference/engine-api/index.md:245]:
```suggestion
curl -X POST --data '{"jsonrpc":"2.0","method":"engine_getBlobsV1","params":[["0x010657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014","0x01f3a5c9d8e7b6a4c2d1e0f9b8a7c6d5e4f3a2b1c0d9e8f7b6a5c4d3e2f1a0b9"]],"id":1}' http://127.0.0.1:8550
```

File: docs/public-networks/reference/engine-api/index.md [hunk 5]
```diff
@@ -224,6 +224,67 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"engine_forkchoiceUpdatedV1","par
...
+
+<TabItem value="wscat WS" label="wscat WS">
+
+```json
+{
+  "jsonrpc": "2.0",
+  "method": "engine_getBlobsV1",
+  "params": [
+    [
+      "0x000657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014",
```
Comment (pinges) [inline docs/public-networks/reference/engine-api/index.md:258]:
```suggestion
      "0x010657f37554c781402a22917dee2f75def7ab966d7b770905398eba3c444014",
```

**Review by pinges — APPROVED** [review]
Please fix the two comments before merging this

File: docs/public-networks/reference/engine-api/objects.md [hunk 6]
```diff
@@ -9,6 +9,15 @@ tags:
 
 The following objects are parameters for or returned by the [Engine API methods](index.md).
 
+## Blob and proof object
+
+Returned by [`engine_getBlobsV1`](index.md#engine_getblobsv1).
+
+| Key | Type | Value |
+| --- | :-: | --- |
+| `blob` | _Data_, 131072 Bytes | An SSZ-encoded `Blob` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844). |
```
Comment (m4sterbunny) [inline docs/public-networks/reference/engine-api/objects.md:18]:
```suggestion
| `blob` | _Data_, 131072 Bytes | An SSZ-encoded `blob` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844). |
```

Comment (m4sterbunny) [reply docs/public-networks/reference/engine-api/objects.md:18]: proper noun for blob?

Comment (m4sterbunny) [reply docs/public-networks/reference/engine-api/objects.md:18]: If proper noun, then consider dropping the ``

**Review by m4sterbunny — APPROVED** [review]
LGTM nitpick flagged

File: docs/public-networks/reference/engine-api/objects.md (hunk 6 above)
Comment (alexandratran) [reply docs/public-networks/reference/engine-api/objects.md:18]: The type alias is capitalized and code formatted according to the linked EIP.

**Pull request merged**